### PR TITLE
Update the project set up information

### PR DESCRIPTION
**Pull request by recomputing**
@joechrysler Take a look and let me know if I missed anything!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,13 +25,13 @@ cd web-dev-interview-2
...
-cd mysite
-
-source bin/activate
+source mysite/bin/activate
 
 pip install Django==1.6.2
 
-django-admin.py startproject mysite --template=https://github.com/chrisdev/foundation-project-zero/zipball/master
+cd mysite
+
```
Comment (recomputing) [inline README.md]: This has the foundation template tags missing from our last round.

File: README.md [hunk 2]
```diff
@@ -25,13 +25,13 @@ cd web-dev-interview-2
...
+django-admin.py startproject --template=https://github.com/chrisdev/foundation-project-account/zipball/master mysite
```
Comment (recomputing) [inline README.md]: This has the foundation template tags missing from our last round.

File: README.md [hunk 3]
```diff
@@ -39,6 +39,11 @@ pip install -r requirements.txt
 
 chmod 755 manage.py
 
+#### Setup the project
+- edit `mysite/settings.py`
+- search for `INSTALLED_APPS`
+- add "mysite" under projects
```
Comment (recomputing) [inline README.md]: Adding mysite app.

**Comment by recomputing** [comment]
@joechrysler I think there was a problem with the db creation, but I can't recreate it.

**Comment by joechrysler** [comment]
@recomputing Looks great! :shipit:

**Pull request merged**